public/src/actions/admin_questions.jsx

```diff
@@ -33,7 +33,27 @@ export function fetchAllQuestions() {
   });
 }
 
-export default function createQuestion(questionInfo, history) {
+export function fetchQuestion(id) {
+  return dispatch => axios.get(`/questions/${id}`, {
+    headers: { Authorization: ` JWT ${localStorage.getItem('auth_token')}` }
+  })
+  .then((res) => {
+    const questionData = res.data;
```

Comment:
consider deconstructing as:
` const { data: questionData } = res `

to remain consistent with other files.